tikv/mvcc.go

```diff
@@ -339,7 +342,7 @@ func (store *MVCCStore) rollbackKey(req *requestCtx, batch *writeBatch, key []by
 	}
 	reader := req.getDBReader()
 	item, err := reader.txn.Get(key)
-	if err != nil {
+	if err != nil && err != badger.ErrKeyNotFound {
```

Comment:
What happend if err is badger.ErrKeyNotFound?

Reply:
Then `item` would be `nil`